### PR TITLE
Corrections to the catalog & ingestion server deployment run books

### PR DESCRIPTION
**Pull request by AetherUnbound**
## Description

This PR makes some corrections to the deployment runbooks for the catalog and the ingestion server, based on observations made during the catalog deployment which occurred earlier today. Namely, this corrects a few links, removes information about the staging catalog (which does not exist), and adds the cloudwatch dashboard deployment instructions.

## Checklist

- [x] My pull request has a descriptive title (not a vague title like`Update index.md`).
- [x] My pull request targets the _default_ branch of the repository (`main`) or a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [x] I added or updated documentation (if applicable).
- [ ] I tried running the project locally and verified that there are no visible errors.
- [ ] I ran the DAG documentation generator (if applicable).

[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Review by zackkrida — APPROVED** [review]
Awesome, thanks for fixing these.

File: documentation/catalog/guides/deploy.md [hunk 1]
```diff
@@ -2,29 +2,43 @@
...
-   1. `just apply prod catalog` and verify the plan before deploying.
+   1. Once you've verified that no DAGs are running, update the value of
+      `running_dags_cleared` to `true` in the
+      [production module declaration](https://github.com/WordPress/openverse-infrastructure/blob/27c41ede9b24991909194e0a6477f6b11fceac0c/environments/prod/catalog-airflow.tf#L33).
+   1. `just apply prod catalog-airflow` and verify the plan before deploying.
+   1. Restore the value of `running_dags_cleared` back to `false`.
+1. Update the Cloudwatch dashboard with the new instance information:
+   1. `just apply prod catalog-dashboard` and verify the plan before deploying
+      (only the catalog EC2 instance ID should be changed within the catalog's
+      dashboard).
```
Comment (sarayourfriend) [inline documentation/catalog/guides/deploy.md:33]: I'll have a PR up momentarily to remove this step. Too tedious to put up with for any longer :joy:

File: documentation/catalog/guides/deploy.md [hunk 2]
```diff
@@ -2,29 +2,43 @@
...
 
 ## Post-deployment steps
 
 1. Check for any Sentry errors in the maintainer's `#openverse-alerts` channel,
    or in the Sentry UI.
+1. Ensure that Airflow is accessible at https://airflow.openverse.engineering.
```
Comment (sarayourfriend) [inline documentation/catalog/guides/deploy.md:39]: Link-like strings aren't automatically converted to `<a>`'s by MyST (IIRC), but wrapping in angle brackets annotates the Markdown properly for it.

```suggestion
1. Ensure that Airflow is accessible at <https://airflow.openverse.engineering>.
```

**Review by sarayourfriend — APPROVED** [review]
LGTM!

File: documentation/catalog/guides/deploy.md (hunk 1 above)
Comment (sarayourfriend) [reply documentation/catalog/guides/deploy.md:33]: https://github.com/WordPress/openverse-infrastructure/pull/571

Comment (zackkrida) [inline documentation/catalog/guides/deploy.md:33]:
```suggestion
```

**Comment by zackkrida** [comment]
@AetherUnbound this needs two tiny revisions (identified via PR review suggestions) and is then ready to merge 😄

**Comment by AetherUnbound** [comment]
For sure, I was planning on correcting it today! I'll draft it until then

File: documentation/catalog/guides/deploy.md (hunk 1 above)
Comment (AetherUnbound) [reply documentation/catalog/guides/deploy.md:33]: (going to roll this into one other change, but thank you!)

**Pull request merged**